ctapipe_io_magic/__init__.py

```diff
@@ -1188,9 +1272,9 @@ def _event_generator(self):
         # Data container - is initialized once, and data is replaced after each yield:
         event = ArrayEventContainer()
 
-        event.meta['origin'] = 'MAGIC'
-        event.meta['max_events'] = self.max_events
-        event.index.obs_id = self.obs_ids[0]
+        event.meta["origin"] = "MAGIC"
+        event.meta["max_events"] = self.max_events
+        event.index.obs_id = self.obs_ids
```

Comment:
Here, it should be:

```bash
event.index.obs_id = self.obs_ids[0]
```

otherwise tools like `cta-process` will not be able to write out columns correctly.

Reply:
For reference:

```bash
ctapipe_io_magic/tests/test_stage1.py::test_stage1_multiple_runs 2023-04-30 12:44:50,621 WARNING [ctapipe.core.telescope_component] (telescope_component.attach_subarray): TelescopeParameter type argument 'MST_*' did not match any known telescope types
2023-04-30 12:44:50,622 WARNING [ctapipe.core.telescope_component] (telescope_component.attach_subarray): TelescopeParameter type argument 'SST_1M_*' did not match any known telescope types
2023-04-30 12:44:50,714 WARNING [ctapipe.ctapipe-process.DataWriter.HDF5TableWriter] (hdf5tableio._create_hdf5_table_schema): Column obs_id with value [5095172] of type <class 'list'>  of container EventIndexContainer in table dl1/event/subarray/trigger not writable, skipping
2023-04-30 12:44:50,717 WARNING [ctapipe.ctapipe-process.DataWriter.HDF5TableWriter] (hdf5tableio._create_hdf5_table_schema): Column obs_id with value [5095172] of type <class 'list'>  of container TelEventIndexContainer in table dl1/event/telescope/trigger not writable, skipping
2023-04-30 12:44:50,719 WARNING [ctapipe.ctapipe-process.DataWriter.HDF5TableWriter] (hdf5tableio._create_hdf5_table_schema): Column obs_id with value [5095172] of type <class 'list'>  of container TelEventIndexContainer in table dl1/event/telescope/parameters/tel_001 not writable, skipping
2023-04-30 12:44:50,728 WARNING [ctapipe.ctapipe-process.DataWriter.HDF5TableWriter] (hdf5tableio._create_hdf5_table_schema): Column obs_id with value [5095172] of type <class 'list'>  of container TelEventIndexContainer in table dl1/event/telescope/images/tel_001 not writable, skipping

```